### DISCUSSION TITLE
Get STW Power level?

### DISCUSSION THREAD

**Discussion by mirronake** [LeleDerGrasshalmi/FortniteEndpointsDocumentation#127] [Q&A] [answered]
Maybe you know how to get that?
![image](https://github.com/LeleDerGrasshalmi/FortniteEndpointsDocumentation/assets/96698262/a43109f8-d0fc-4e0e-902e-96b6c62b4b68)

**Comment by taradinoc** [comment, marked as answer]
To calculate homebase power level, you need to fetch the profile (`QueryPublicProfile` endpoint), calculate its FORT stats based on its slotted survivors and research upgrades, then convert the FORT stats to power level. See https://github.com/fnbrjs/fnbr.js/blob/0cfe1edacdd185c10ca05c837e6e9fb989b7b167/src/structures/stw/STWProfile.ts#L235-L242 for an implementation.
